### DISCUSSION TITLE
Missing @parcel/transformer-inline-string in new beta

### DISCUSSION THREAD

**Discussion by eirikb** [parcel-bundler/parcel#6001] [General] [answered]
Hi

When I run `npm install parcel@next` I get the new **beta.2**. But I don't get all the dependencies, such as `@parcel/transformer-inline-string`.
I see the dependency here: https://www.npmjs.com/package/@parcel/transformer-inline-string/v/2.0.0-beta.2 .
But it is missing from https://www.npmjs.com/package/@parcel/config-default/v/2.0.0-beta.2

Example:
![image](https://user-images.githubusercontent.com/241706/111292982-0fc16800-8649-11eb-80c4-bbd8bca5437e.png)

**Comment by devongovett** [comment, marked as answer]
We removed some less commonly used plugins from the default config in order to reduce npm install size. These will usually be automatically installed into your project (eg when you use the bundle-text: pipeline). If you have autoinstall turned off you'll need to install them locally yourself.

**Reply by eirikb** [reply]
Thank you for the answer :)

Note:
  * The automatically installed version of @parcel/transformer-inline-string was "^2.0.0-alpha.3", even when parcel was set without "^".
  * Version of @parcel/transformer-sass (random  dep I tested) became "^2.0.0-beta.2" (more correct).
  *  The node_modules/@parcel/transformer-inline-string folder is 32K on my computer, and node_modules will be 124K when installing fresh (but this includes dependencies that would be there anyway), see image below.
  * @parcel/transformer-sass I can understand, this ends up being 7.6M

Now I know the reason, that is all I need.

![image](https://user-images.githubusercontent.com/241706/111326344-c551e280-866c-11eb-9cbb-3565b0c85f3b.png)
